config/project.cmake

```diff
@@ -65,6 +65,8 @@ set_property(CACHE FLECSI_RUNTIME_MODEL
 #------------------------------------------------------------------------------#
 
 # This changes the Cinch default
+# Note: This has to be added in projects that include flecsi too because it
+# doesn't persist.
```

Comment:
That is a bit stupid...

If flecsi is a subproject its `project.cmake` _of course_ doesn't get sourced and hence cinch's default, which is off is used. I guess that should really go into `flecsi.h` (#196).